zebra-state/src/service/read/difficulty.rs

```diff
@@ -308,16 +319,28 @@ fn adjust_difficulty_and_time_for_testnet(
         .expect("a valid block time minus a small constant is in-range");
 
     if result.cur_time <= local_std_difficulty_limit {
-        // Standard difficulty: the max time needs to exclude min difficulty blocks
```

Comment:
We may want to update this comment a few lines above:
> - if cur_time gets clamped to max_time, this is already a minimum difficulty block.

for when `max_time` from `median_time_past` is behind the `std_difficulty_max_time` from the previous block time.

This wasn't changed in this PR but I thought the condition was `result.cur_time <=  min_difficulty_min_time`, why subtract `POST_BLOSSOM_POW_TARGET_SPACING * 2` from the `std_difficulty_max_time`?

Reply:
The miner is going to take approximately ~75 seconds to mine the next block from this template, so there's no point starting a difficult block if we're close to doing an easy one.

I'll make a constant for this.

Reply:
Fixed in 9842f52